.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Release
+about: When ready to cut a release
+title: Release X.Y.Z
+labels: ''
+assignees: ''
+
+---
+
+- [ ] Create release branch:
+```bash
+$ git flow release start X.Y.Z
+```
+- [ ] Edit `setup.py` to reflect the version of this release
+- [ ] Bump spec version (if applicable)
```

Comment:
I think that we should consider adding a step under this to ensure that the outstanding changes are committed. Something like:

```
$ git add .
$ git commit -m "X.Y.Z"
```